driver/modern_bpf/programs/tail_called/events/syscall_dispatched_events/setpgid.bpf.c

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (C) 2022 The Falco Authors.
+ *
+ * This file is dual licensed under either the MIT or GPL 2. See MIT.txt
+ * or GPL2.txt for full copies of the license.
+ */
+
+#include <helpers/interfaces/fixed_size_event.h>
+
+/*=============================== ENTER EVENT ===========================*/
+
+SEC("tp_btf/sys_enter")
+int BPF_PROG(setpgid_e,
+	     struct pt_regs *regs,
+	     long id)
+{
+	struct ringbuf_struct ringbuf;
+	if(!ringbuf__reserve_space(&ringbuf, SETPGID_E_SIZE))
+	{
+		return 0;
+	}
+
+	ringbuf__store_event_header(&ringbuf, PPME_SYSCALL_SETPGID_E, SETPGID_E_SIZE);
+
+	/*=============================== COLLECT PARAMETERS  ===========================*/
+
+	/* Parameter 1: pid (type: PT_FD) */
+	pid_t pid = (s32)extract__syscall_argument(regs, 0);
+	ringbuf__store_s64(&ringbuf, (s64)pid);
+
+	/* Parameter 2: pgid (type: PT_PID) */
+	pid_t pgid = (s32)extract__syscall_argument(regs, 1);
+	ringbuf__store_s64(&ringbuf, (s64)pgid);
+
+	/*=============================== COLLECT PARAMETERS  ===========================*/
+
+	ringbuf__submit_event(&ringbuf);
+
+	return 0;
+}
+
+/*=============================== ENTER EVENT ===========================*/
+
+/*=============================== EXIT EVENT ===========================*/
+
+SEC("tp_btf/sys_exit")
+int BPF_PROG(setpgid_x,
+	     struct pt_regs *regs,
+	     long ret)
+{
+	struct ringbuf_struct ringbuf;
+	if(!ringbuf__reserve_space(&ringbuf, SETPGID_X_SIZE))
+	{
+		return 0;
+	}
+
+	ringbuf__store_event_header(&ringbuf, PPME_SYSCALL_SETPGID_X, SETPGID_X_SIZE);
+
+	/*=============================== COLLECT PARAMETERS  ===========================*/
+
+	/* Parameter 1: res (type: PT_PID)*/
```

Comment:
PT_ERRNO?

Reply:
In the event table, we have used `PT_PID`  https://github.com/falcosecurity/libs/blob/2fa7b0dedd0c87d53fb46c5e264b94bdf47b8d19/driver/event_table.c#L310 so right now i will keep it like it is, but yeah we need to find a good convention! probably we will work on that here https://github.com/falcosecurity/libs/pull/526